{{cookiecutter.project_name}}/recipe.yaml

```diff
@@ -85,5 +85,5 @@ metrics:
       function: weighted_mean_squared_error
       greater_is_better: False
   # Sets the primary metric to use to evaluate model performance. This primary metric is used
-  # to sort MLflow Runs corresponding to the pipeline in the MLflow Tracking UI
+  # to sort MLflow Runs corresponding to the recipe in the MLflow Tracking UI
```

Comment:
Unfortunately I can't make the comment on other files but can we fix the recipe.yaml with the new product spec?

Example in recipe:

`metrics` are renamed to `custom_metrics`
We don't have `data_scoring` step anymore, etc

The new spec can we found here: https://github.com/mlflow/mlp-regression-example/pull/14/files#diff-1250fe52790be66c0718296d26be97fdaee7cfca04d0448f5b87dba907dfa669

Reply:
It worth a different PR to re-copy the new recipe file once it's released.

Reply:
Nevermind about the "released". I'll have a followup PR to address this.